testing/dbtest/docs.go

```diff
@@ -0,0 +1,22 @@
+// Package dbtest provides a way to create a clean database for tests using template databases.
+// For it to function properly a postgres database server must be running with the expected templates.
```

Comment:
Really long lines again.. feel free to ignore this but it's a bit jarring compared with others.

Reply:
fixed